subsys/bluetooth/host/conn.c

```diff
@@ -1812,6 +1812,10 @@ int bt_conn_get_info(const struct bt_conn *conn, struct bt_conn_info *info)
 		info->le.interval = conn->le.interval;
 		info->le.latency = conn->le.latency;
 		info->le.timeout = conn->le.timeout;
+		info->le.features = conn->le.features;
+#if defined(CONFIG_BT_TESTING)
+		info->rv = &conn->rv;
+#endif /* CONFIG_BT_TESTING */
 		return 0;
 #if defined(CONFIG_BT_BREDR)
 	case BT_CONN_TYPE_BR:
```

Comment:
explicitly set info->rv = NULL here?
We don't initiate read_remote_version for BR/EDR connections.